app/[...slug]/page.tsx

```diff
@@ -65,8 +65,9 @@ export async function generateMetadata({ params }: { params: ParamsType }): Prom
 export default async function Page({ params }: { params: ParamsType }) {
   const { data } = await getStoryData(params);
   const bridgeOptions = { resolveRelations };
+  let slug: string = params.slug ? params.slug.join('/') : '';
 
   return (
-    <StoryblokStory story={data.story} bridgeOptions={bridgeOptions} />
+    <StoryblokStory story={data.story} bridgeOptions={bridgeOptions} slug={slug} />
```

Comment:
In the future we might want to access the slug in in the stories so passing this in. I've tested that this works.

Also, this route [...slug] is for non-homepage so I'm removing the "home" condition.